core/src/test/scala/org/broadinstitute/dsde/rawls/entities/datarepo/DataRepoBigQuerySanitizationSpec.scala

```diff
@@ -0,0 +1,70 @@
+package org.broadinstitute.dsde.rawls.entities.datarepo
+
+import java.nio.charset.Charset
+
+import com.google.common.io.Resources
+import org.scalatest.{BeforeAndAfterAll, FreeSpec}
+import spray.json._
+
+class DataRepoBigQuerySanitizationSpec extends FreeSpec with DataRepoBigQuerySupport with BeforeAndAfterAll {
+
+  // read the list of naughty strings from https://github.com/minimaxir/big-list-of-naughty-strings.
+  // We expect a json array, containing base64-encoded strings.
+  val blnsData = Resources.toString(
+    new java.net.URL("https://raw.githubusercontent.com/minimaxir/big-list-of-naughty-strings/master/blns.base64.json"),
+    Charset.defaultCharset())
```

Comment:
reviewers: how do you feel about this test relying on downloading a third-party resource from github? we already rely on downloading third-party jars, etc. to compile, so it feels safe. I like downloading it each time, so that if it gets updated with new test cases, we automatically adopt those test cases.

Reply:
given that in the unit test phase we already rely on github to download the code in the first place I am ok with the github dependency. I am a little concerned about depending on an externally controlled file that may be updated and break our tests at any time.

Reply:
Also agreed that it should generally be fine. 

In the rare case it disappears, it would a small fire - so maybe adding on something later to make this resilient to the json disappearing altogether (maybe our own list of strings) might be useful. 

Agreed that it should be allowed to fail if an update of the json to add more strings breaks our code.